releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -495,7 +517,7 @@
   "123105": {
     "commit": "b2c32fb0151cb56fa30a9cc60b0835ed5c8dfcd5",
     "text": "The kube-proxy nodeportAddresses / --nodeport-addresses option now\naccepts the value \"primary\", meaning to only listen for NodePort connections\non the node's primary IPv4 and/or IPv6 address (according to the Node object).\nThis is strongly recommended, if you were not previously using\n--nodeport-addresses, to avoid surprising behavior.\n\n(This behavior is enabled by default with the nftables backend; you would\nneed to explicitly request `--nodeport-addresses 0.0.0.0/0,::/0` there to get\nthe traditional \"listen on all interfaces\" behavior.)",
-    "markdown": "The kube-proxy nodeportAddresses / --nodeport-addresses option now\n  accepts the value \"primary\", meaning to only listen for NodePort connections\n  on the node's primary IPv4 and/or IPv6 address (according to the Node object).\n  This is strongly recommended, if you were not previously using\n  --nodeport-addresses, to avoid surprising behavior.\n  \n  (This behavior is enabled by default with the nftables backend; you would\n  need to explicitly request `--nodeport-addresses 0.0.0.0/0,::/0` there to get\n  the traditional \"listen on all interfaces\" behavior.) ([#123105](https://github.com/kubernetes/kubernetes/pull/123105), [@danwinship](https://github.com/danwinship)) [SIG API Machinery, Network and Windows]",
+    "markdown": "The kube-proxy nodeportAddresses / --nodeport-addresses option now\n  accepts the value \"primary\", meaning to only listen for NodePort connections\n  on the node's primary IPv4 and/or IPv6 address (according to the Node object).\n  This is strongly recommended, if you were not previously using\n  --nodeport-addresses, to avoid surprising behavior.\n  \n  (This behavior is enabled by default with the nftables backend; you would\n  need to explicitly request `--nodeport-addresses 0.0.0.0/0,::/0` there to get\n  the traditional \"listen on all interfaces\" behavior.) ([#123105](https://github.com/kubernetes/kubernetes/pull/123105), [@danwinship](https://github.com/danwinship))",
```

Comment:
```suggestion
    "markdown": "The kube-proxy nodeportAddresses / --nodeport-addresses option now\n  accepts the value \"primary\", meaning to only listen for NodePort connections\n  on the node's primary IPv4 and/or IPv6 address (according to the Node object).\n  This is strongly recommended, if you were not previously using\n  --nodeport-addresses, to avoid surprising behavior.\n  \n  (This behavior is enabled by default with the nftables backend; you would\n  need to explicitly request `--nodeport-addresses 0.0.0.0/0,::/0` there to get\n  the traditional \"listen on all interfaces\" behavior.) ([#123105](https://github.com/kubernetes/kubernetes/pull/123105), [@danwinship](https://github.com/danwinship)) [SIG API Machinery, Network and Windows]",
```